demos/applications/ws2812b.cpp

```diff
@@ -0,0 +1,138 @@
+#include <array>
+#include <span>
+
+#include <libhal-util/serial.hpp>
+#include <libhal-util/steady_clock.hpp>
+
+#include <libhal-display/ws2812b.hpp>
```

Comment:
```suggestion
#include <libhal-util/serial.hpp>
#include <libhal-util/steady_clock.hpp>
#include <libhal-display/ws2812b.hpp>
```

Keep these together and run clang-format on this file.